components/button_group/button_group_default.story.vue

```diff
@@ -1,14 +1,26 @@
 <template>
   <dt-button-group
     :alignment="alignment"
+    class="d-gg8"
   >
     <html-fragment
       v-if="defaultSlot"
       :html="defaultSlot"
     />
+    <template v-if="alignment === 'end'">
```

Comment:
`v-else-if` since the slot is meant to override the default example, rather than add markup beside it.

Reply:
Fixed! Thanks 🙌